### DISCUSSION TITLE
CLI: add "clear-cache" command

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#16694] [Feature Requests]
In order to clear the cache from a deployment pipeline I'd suggest to add a clear-cache command into the CLI. This could be run before each server-start to make sure all changes are catched-up correctly.

**Comment by rijkvanzanten** [comment]
Flushes are cached on server startup regardless 🙂 

https://github.com/directus/directus/blob/dda1afcf16f95e528c41cd0767b3b4fafb8de5ac/api/src/app.ts#L87

That being said, could still be a useful cli command to have. In the meantime, you could also rely on the cache clear endpoint: https://docs.directus.io/reference/system/utilities.html#clear-the-internal-cache

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
